XamarinCommunityToolkit/Views/SideMenuPosition.shared.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace Microsoft.Toolkit.Xamarin.Forms.UI.Views
```

Comment:
Could this class not be combined with `SideMenuState` somehow? It's essentially the same enum.

Reply:
Yeah, I thought about it... But they have totally different sense.

SideMenuState is a current state of the menu (Left/Right menu opened or not)
SedeMenuPosition is a position of the view in the side menu control (Is it Left/Right menu or not)

I afraid to bring a mess

@jfversluis

Reply:
Left as is